gen-ai/inference/stable-diffusion-rayserve-gpu/ray-serve-stablediffusion.yaml

```diff
@@ -63,8 +64,8 @@ spec:
             # Notice that we are using the same image for both the head and worker nodes. You might hit ModuleNotFoundError if you use a different image for head and worker nodes.
             # Preload Container Image into data volumes for faster new ray worker nodes
             - name: head
-              image: public.ecr.aws/data-on-eks/ray2.11.0-py310-gpu-stablediffusion:latest
-              imagePullPolicy: IfNotPresent # Ensure the image is always pulled when updated
+              image: public.ecr.aws/data-on-eks/ray-serve-gpu-stablediffusion:2.33.0-py311-gpu
+              imagePullPolicy: Always # Ensure the image is always pulled when updated
```

Comment:
Why `Always`?

Reply:
If the image public.ecr.aws/data-on-eks/ray-serve-gpu-stablediffusion:2.33.0-py311-gpu is not up-to-date, then kubelet will pull the image from ECR. Otherwise, it will use the cached image. 

https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy

Always
every time the kubelet launches a container, the kubelet queries the container image registry to resolve the name to an image digest. If the kubelet has a container image with that exact digest cached locally, the kubelet uses its cached image; otherwise, the kubelet pulls the image with the resolved digest, and uses that image to launch the container.

Reply:
Thus, if I update the image with the same tag, I can make sure it will use the latest version.  If the SHA256 is the same, then kubelet will use the cached image on disk